app/routes.py

```diff
@@ -0,0 +1,279 @@
+from app import db 
+from flask import Blueprint, jsonify, request, make_response 
+from app.models.customer import Customer
+from app.models.video import Video
+from app.models.rental import Rental
+from datetime import datetime, timedelta
+import requests
+import os
+from dotenv import load_dotenv
+
+customers_bp = Blueprint("customers",__name__,url_prefix = "/customers") # path that gives you access to resources/endpoint
+videos_bp = Blueprint("videos",__name__,url_prefix = "/videos")
+rentals_bp = Blueprint("rentals",__name__,url_prefix = "/rentals")
+
+@videos_bp.route("", methods=["GET", "POST"])
+def handle_videos():
+    if request.method == "POST":
+        request_body = request.get_json()
+        if "title" not in request_body: 
+            return jsonify({"details":"Request body must include title."}), 400
+        if "release_date" not in request_body: 
+            return jsonify({"details":"Request body must include release_date."}), 400
+        if "total_inventory" not in request_body: 
+            return jsonify({"details":"Request body must include total_inventory."}), 400
+        else:
+            new_video = Video(
+            title= request_body["title"],
+            release_date=request_body["release_date"],
+            total_inventory =request_body["total_inventory"])
+
+        db.session.add(new_video)
+        db.session.commit()
+        new_video_response = {
+                "id": new_video.id,
+                "title" : new_video.title,
+                "release_date": datetime.now(),
+                "total_inventory": new_video.total_inventory
```

Comment:
this repeated code could be moved to your Video Model as an instance method or moved to a helper function. It could look something like this after the instance method is created
```suggestion
                video.to_video_dict()
```